### ISSUE TITLE
Printing images

### ISSUE THREAD

**Issue by trobrock** [opentable/star-printing#2]
Is is possible to print images on receipts using this library?

**Comment by deepesh259nitk** [comment]
Yes star micronics SDK supports images as well : refer documentation : https://www.starmicronics.com/support/sdkdocumentation.aspx
